src/bastion/examples/round_robin_dispatcher.rs

```diff
@@ -0,0 +1,106 @@
+use bastion::prelude::*;
+use std::sync::Arc;
+
+///
+/// Prologue:
+/// This example demonstrate a idiomatic way to implement the round robin
+/// algorithm with bastion. We will use two groups of children, one will be
+/// supervised by a dispatcher, named `Rounder`, the other one will call it.
+/// Both groups will be supervised by the supervisor.
+///
+///         Bastion
+///            |
+///        Supervisor     
+///       /          \
+///    Caller      Rounder
+///      |            |
+///   Children     Children
+///
+/// 1. We want a group of children which will broadcast a message to a defined
+/// target with a defined data.
+/// 2. We want a group of children for receive and print the message.
+/// 3. We want to use a dispatcher on the second group because we don't want to
+/// target a particular child in the first to process the message.
+///
+fn main() {
+    // We need bastion to run our program
+    Bastion::init();
+    // We create the supervisor and we add both groups on it
+    Bastion::supervisor(caller_supervisor)
+        .and_then(|_| Bastion::supervisor(rounder_supervisor))
```

Comment:
rounder sounds a bit confusing to me, would there be an other name for this ? maybe Callee, or Receiver?

Reply:
It make sense yes.